docs/signals/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Snowplow Signals"
+sidebar_position: 8
+description: "An overview of Signals concepts."
+sidebar_label: "Signals"
+---
+
+Snowplow Signals empowers users to build more intelligent AI applications that can perceive and act on customer behavior in real-time. By aggregating engineer-defined attributes and facts, Signals enables Large Language Models (LLMs) to act intelligently and deliver personalized experiences.
```

Comment:
What does "engineer-defined" mean?  Perhaps "user-defined" vs pulling in an engineer...